CONTRIBUTORS.md

```diff
@@ -16,3 +16,4 @@
 ## Matchmaking Project
 
 - [Example](https://github.com/phil-ociraptor)
+- [Kenneth Lee] (https://github.com/Leeoku)
```

Comment:
For markdown (`.md`) files to render links, there can be no space between the brackets and the parenthesis

Reply:
Space removed! Curious question, will github know to auto put the code even if this PR was behind van's?